### PR TITLE
Update nodejs-sdk.md

### PR DESCRIPTION
**Pull request by sbeskin**

### CODE REVIEW

File: docs/documentation/nodejs-sdk.md [hunk 1]
```diff
@@ -276,14 +276,18 @@ const whitelistedTransaction = account.whitelistTransaction({ envelope: clientTr
...
 Note that if you are whitelisted, any payment sent from a server developed with the Node SDK is already considered whitelisted, so the server transactions will not need the above step.
 
-#### Decode transaction
-When the client sends you a transaction for whitelisting, it will be encoded. If you wish to decode the transaction and verify its details before whitelisting it:
+#### Decode Transaction
+When the client sends you a transaction for whitelisting, it will be XDR-encoded. You may want to decode the transaction and verify its details before whitelisting it. There are two methods of decoding an XDR-encoded transaction:
+
+- Decode only to "RawTransaction": This method returns a "RawTransaction" object. It contains the  entire transaction data, including some fields that are of no use to the user.
+
+  ```const transaction = Transaction.decodeRawTransaction({ envelope: encodedTransaction, networkId: Network.current().networkPassphrase() });```
```
Comment (cnotethegr8) [inline docs/documentation/nodejs-sdk.md]: @sbeskin when using a multiline code block (\```) please follow the standard of having the \` on separate lines, and using the language name for proper syntax highlighting.

Here is an example. https://help.github.com/en/articles/creating-and-highlighting-code-blocks#syntax-highlighting

The name you want to use is, javascript

**Review by cnotethegr8 — CHANGES_REQUESTED** [review]

File: docs/documentation/nodejs-sdk.md (hunk 1 above)
Comment (yosriz) [reply docs/documentation/nodejs-sdk.md]: @ravitb also reformat the param object, so it will be in a new line for better readability

**Review by cnotethegr8 — COMMENTED** [review]
@ravitb you should replace ` ``` ` with ` ```javascript` for proper syntax highlighting.

File: docs/documentation/nodejs-sdk.md (hunk 1 above)
Comment (ravitb) [reply docs/documentation/nodejs-sdk.md]: @cnotethegr8 please notice that Sasha fixed the comment. Please approve it.

**Review by cnotethegr8 — APPROVED** [review]

**Pull request merged**